### PR TITLE
SUPPORT.md: HTTP => HTTPS

### PR DESCRIPTION
**Pull request by Schweinepriester**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:

- Skipping the redirect HTTP => HTTPS this way for http://stackoverflow.com/help/on-topic 0:-)
- http://slack.k8s.io isn't being redirected even :o

#### Which issue(s) this PR fixes:

Fixes #N/A

#### Special notes for your reviewer:

:)

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: SUPPORT.md [hunk 1]
```diff
@@ -20,10 +20,10 @@ The Kubernetes Community is active on Stack Overflow, you can post your question
...
-* [Slack](https://kubernetes.slack.com) ([registration](http://slack.k8s.io)):
+* [Slack](https://kubernetes.slack.com) ([registration](https://slack.k8s.io)):
 The `#kubernetes-users` and `#kubernetes-novice` channels are usual places where
 people offer support.
 
 ### Forum
 
-* [Kubernetes Official Forum](https://discuss.kubernetes.io)
\ No newline at end of file
+* [Kubernetes Official Forum](https://discuss.kubernetes.io)
```
Comment (aojea) [inline SUPPORT.md:29]: what has changed here?

Comment (Schweinepriester) [reply SUPPORT.md:29]: I'm actually not sure and wondered myself!

I used the GitHub Editor (like, clicking the edit button next to a Markdown file) to edit the two URLs and only when I checked in the PR I noticed this line is marked as changed as well. Sometimes the editor is inserting an empty newline at the end, but that doesn't seem to be the case here?
Maybe it converted some non-LF line ending to LF or something? 🤔

Comment (Schweinepriester) [reply SUPPORT.md:29]: It's the newline, never mind:

```bash
@Schweinepriester ➜ /workspaces/kubernetes (patch-1) $ git show a0d6a49b12e0bffd13f6a499037f81b49b705c98
// [TRUNCATED…]
-* [Kubernetes Official Forum](https://discuss.kubernetes.io)
\ No newline at end of file
+* [Kubernetes Official Forum](https://discuss.kubernetes.io)
```

**Review by mrbobbytables — COMMENTED** [review]
/sig contributor-experience
/triage accepted
/lgtm
/assign @thockin

**Comment by thockin** [comment]
Thanks!

/lgtm
/approve

**Pull request merged**